api-specificatie/Bewoning/openapi.yaml

```diff
@@ -19,7 +17,7 @@ paths:
+      description: "<body><p><body> <p>Het raadplegen van een Bewoning. <br> Een bewoning is een verblijfplaats waar minimaal 1 persoon is ingeschreven. Op basis van de identificatiecodenummeraanduiding retourneert Bewoning de personen die op de verblijfplaats zijn ingeschreven.</p> <p><br />Op basis van de parameters wordt een of meerdere bewoningen geretourneerd inclusief gegevens over de bewoners van het betreffende verblijfobject. Ten minste &eacute;&eacute;n van de volgende parameters moet zijn opgenomen.<br /></p><ol><li> Identificatiecodenummeraanduiding <br /></li> <li> Burgerservicenummer </li> </ol> </p><p><br>De bovenstaande parameters moeten gecombineerd worden met een peildatum of een periode.</p> <p>Overleden personen worden opgenomen in het zoekresultaat tot aan de datum van overlijden voor zover deze datum overlijden binnen de opgevraagde periode valt.</p> <p>Het resultaat bevat de adressen en bewoners van een verblijfplaats waarvan de EINDdatum geldigheidverblijfplaats op of na de <i>datumvan</i> ligt EN de BEGINdatum op of voor de <i>datumtotenmet</i> ligt.</p> <p>Of</p> <p>Het resultaat bevat de het adres en de bewoners van de verblijfplaats waarbij de BEGINdatumgeldigheidverblijfplaats voor of op de <i>peildatum</i> ligt en de EINDdatumgeldigheidverblijfplaats na of op de <i>peildatum</i> ligt.</p> <p>Er wordt niet op de gevonden adressen gesorteerd, maar per adres (bewoning) worden de bewoners wel gesorteerd op datumvan.</p></li></ol></body>"
```

Comment:
Waarom is er eigenlijk voor gekozen om de html in de description te gebruiken? De elementen die gebruikt worden kunnen ook met markdown worden gerealiseerd. En als er html gebruikt wordt moet de tags goed gecontroleerd worden op closing tags. Als je in de eerste regel kijk, is er al een html fout namelijk twee <body> tags

Reply:
We hebben destijds bij het genereren van specificaties voor html gekozen. Dat werkt ook en gaf wat minder issues dan Markdown.  Daarna zijn er wijzigingen in de descriptions doorgevoerd en daarbij zijn de fouten in de html erin geslopen . 
Ik heb de dubbele body tag verwijderd.